app/forms/MovementForms.scala

```diff
@@ -32,13 +32,26 @@ object ChoiceForm {
   implicit val format = Json.format[ChoiceForm]
 
   //TODO change to enum
-  private val correctChoice = Seq("EAL", "EDL")
+  private val correctChoice = Seq("SMP", "STD", "EAL", "EDL")
 
   val choiceMapping = mapping(
     "choice" -> text().verifying("Incorrect value", correctChoice.contains(_))
   )(ChoiceForm.apply)(ChoiceForm.unapply)
 }
 
+case class RoleForm(roleForm: String)
+
+object RoleForm {
+  implicit val format = Json.format[RoleForm]
+
+  //TODO change to enum
```

Comment:
Not enum, it can be sealed trait and case classes, enum is not good :) I think we can stay with `Seq`. It can be `Set` too